### PR TITLE
LTD-4142-update-test-to-confirm-finalisation-no-inform

### PR DESCRIPTION
**Pull request by depsiatwal**
This Updates the tests during refusal finalisation to ensure we can proceed with just a refusal letter

https://uktrade.atlassian.net/browse/LTD-4142

### CODE REVIEW

File: api/cases/tests/test_refuse_licence.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ def test_refuse_standard_application_success(self, send_exporter_notifications_f
         for document in GeneratedCaseDocument.objects.filter(advice_type__isnull=False):
             self.assertTrue(document.visible_to_exporter)
 
+        # Confirm finalisation can succeed with only a refuse doc
+        self.assertEqual(GeneratedCaseDocument.objects.filter(case=self.application.id)[0].advice_type, "refuse")
```
Comment (kevincarrogan) [inline api/cases/tests/test_refuse_licence.py:44]: This says "with only a refuse doc" but then the assertion doesn't check that there's _only_ a refuse doc, it just checks that the _one_ of the documents is a refusal.

Comment (depsiatwal) [reply api/cases/tests/test_refuse_licence.py:44]: good point need to make sure only one exists

Comment (depsiatwal) [reply api/cases/tests/test_refuse_licence.py]: done

**Review by kevincarrogan — APPROVED** [review]

**Review by Tllew — APPROVED** [review]
LGTM

**Pull request merged**